lib/pbench/test/unit/server/test_server_audit.py

```diff
@@ -1,19 +1,30 @@
 import datetime
 from http import HTTPStatus
-from typing import Optional
+from typing import Callable, Optional
 
 import dateutil.parser
+from flask import Response
+from flask.testing import FlaskClient
 from freezegun.api import freeze_time
 import pytest
 import requests
 
-from pbench.server import JSONOBJECT, OperationCode
-from pbench.server.database.models.audit import Audit, AuditStatus
+from pbench.server import JSONOBJECT, OperationCode, PbenchServerConfig
+from pbench.server.database.models.audit import Audit, AuditReason, AuditStatus
+from pbench.server.database.models.datasets import Dataset
+from pbench.server.database.models.users import User
 
 
 class TestServerAudit:
+    audits = []
+
     @pytest.fixture()
-    def query_get(self, client, server_config, pbench_admin_token):
+    def query_get(
+        self,
+        client: FlaskClient,
```

Comment:
This is actually a bit intriguing. I added a reference to `AuditReason` and normally `vscode` is quite obliging on a `ctrl-.` to add imports, but didn't see it. It *did* have an option I'd never noticed before to add types for all fixtures (yes, specifically "fixtures"), and I accidentally let go of the button hovering over it. So it went ahead. And I had to search for how to import a few (like `FlaskClient`, which isn't trivially obvious), but I decided to leave it...